rest/route/host.go

```diff
@@ -441,12 +441,32 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 		toTerminate.TerminatedVolumes = append(toTerminate.TerminatedVolumes, v.ID)
 	}
 
+	if !ch.dryRun {
+		if err := ch.clearLogin(); err != nil {
+			catcher.Wrapf(err, "clearing user login cache")
+		}
+	}
+
 	if catcher.HasErrors() {
 		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(catcher.Resolve(), "not all unexpirable hosts/volumes terminated"))
```

Comment:
I think this error wrapper is helpful but this whole chunk should probably be above your section, so it applies only to the terminating hosts/volumes section, probably clearing the catcher and adding this new verbose error, and then also adding the usr login cache error (otherwise if the only error is the login cache it'll read "not all unexpirable hosts/volumes terminated: error clearing user login cache")